src/parsing/fol/grammar.pest

```diff
@@ -126,11 +126,16 @@ annotated_formula_eoi = _{ annotated_formula ~ EOI }
 specification = { (annotated_formula ~ ".")* }
 specification_eoi = _{ specification ~ EOI }
 
+placeholder_sort = _{ general | symbol | integer }
+    general = { "general" }
+    symbol  = { "symbol" }
+    integer = { "integer" }
```

Comment:
Do we want to have a special placeholder sort or should we recycle our normal sort?